app/components/view-section-table-row/template.njk

```diff
@@ -15,6 +15,8 @@
     {% endverbatim %}
   </code></pre>
 
+  {% set commonAttributes %} data-test-id="view-section-table-row-{{ params.questionId }}" class="nhsuk-width-container bc-c-section-table-row"{%- endset -%}
```

Comment:
what's this about? `commonAttributes ` and where is it used?

Reply:
oh yeah I left that there by mistake 👀 good spot